### DISCUSSION TITLE
Spectrum News NY1

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#760] [Q&A]
### Channel Name

Spectrum News NY1

### Channel ID

SpectrumNewsNY1.us

### Country

United States

### Language

English

### Website

https://www.ny1.com/nyc/all-boroughs

**Comment by what-a** [comment]
Suggested logo:
https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/Spectrum_News_NY1.svg/320px-Spectrum_News_NY1.svg.png

![image](https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/Spectrum_News_NY1.svg/320px-Spectrum_News_NY1.svg.png)

**Comment by Central302** [comment]
[Note]Cable Access Required.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19765
